wren-ui/src/components/pages/setup/dataSources/BigQueryProperties.tsx

```diff
@@ -52,7 +57,10 @@ const UploadCredentials = (props) => {
   );
 };
 
-export default function BigQueryProperties() {
```

Comment:
We need to deal with the `DuckDBProperties`, right?

Reply:
Nope. All DuckDB properties can be updated.